bba/bba.go

```diff
@@ -511,3 +563,28 @@ func (bba *BBA) processAuxMessage(msg *pb.Message_Bba) (cleisthenes.Request, uin
 	}
 	return req, msg.Bba.Round, nil
 }
+
+func (bba *BBA) processDelayedMessage(round uint64, sender cleisthenes.Member, req cleisthenes.Request) {
```

Comment:
how about `saveIncomingRequest`?

Reply:
I think it is better to expose "it is delayed message". how about `saveDelayedMessage`?

Reply:
what it's function do is `save` and in the context of current round of bba, it is not delayed but future request so I suggest to use word `incoming`